modules/aws/volume-access/variables.tf

```diff
@@ -0,0 +1,75 @@
+variable "sn_policy_version" {
+  description = "The value of SNVersion tag"
+  default     = "3.16.1" # {{ x-release-please-version }}
+  type        = string
+}
+
+variable "region" {
+  default     = "*"
+  description = "The AWS region where your instance of StreamNative Cloud is deployed. Defaults to all regions \"*\""
+  type        = string
+}
+
+variable "streamnative_vendor_access_role_arns" {
+  default     = ["arn:aws:iam::311022431024:role/cloud-manager"]
+  description = "This role for access customer s3 bucket on control plane."
+  type        = list(string)
+}
+
+variable "additional_federated_identifiers" {
+  default     = []
+  description = "This federated identified list for access customer s3 bucket on data plane."
+  type        = list(string)
+}
```

Comment:
This variable is not been used by anywhere, Let's remove it.
```suggestion
```

Reply:
Fixed